Technologies/tanzil.md

```
@@ -0,0 +1,161 @@
+    You can find all of these translations here: [https://tanzil.net/trans/](https://tanzil.net/trans/)
+
+
+## Source Code
+
+Tanzil does not seem to be an open source project since it source code is not found on their website and can not be found on the internet search engines.
+
+## Resources
+
+Tanzil clarified that it gets many of its data from different resousrces, and part of it by their own effort.
+
+ - **Quran Text**: They build different applications to extract quran text, review it, and build the database. After the extraction, they verify manually and examined it thoroughly against Medina Mushaf that is provided by [King Fahd Quran Complex](https://qurancomplex.gov.sa/), and a checksum for each letter and diacritic was calculated to verify the accuracy of the text.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)